src/components/DatePicker/DatePicker.tsx

```diff
@@ -46,12 +49,13 @@ export default function DatePicker(props: Props): JSX.Element {
     </>
   );
 
-  // set default timezone to UTC
-  moment.tz.setDefault('UTC');
+  // set timezone, defaulting to browser
+  const browserTimeZone = Intl.DateTimeFormat(locale).resolvedOptions().timeZone;
```

Comment:
We need UTC as default for now otherwise it will break production.

Reply:
Or we could update all datepickers to pass UTC as default time zone

Reply:
I'll go ahead and default to UTC 👍